data/competencies.yml

```
@@ -70,6 +72,10 @@
     - Fixes vulnerabilities in dependencies raised by Snyk
     - Fixes vulnerabilities raised by the cyber security team
     - Doesn't introduce vulnerabilities outlined in the Open Web Application Security Project (OWASP) top 10
+    - EUC Endpoint or Server Patching. 
+    - Maintains Anti-Virus.ervices 
+    - Active Directory Group Policy Review and Hardening
+    - Improves the SRS score
```

Maybe later we should add a supporting URL to this as well?
```suggestion
    - Improves the Security Risk Score
```